test/unit_tests/test_rules.py

```diff
@@ -82,17 +82,6 @@ def test_eval_match_file_measurements():
     result = rules.eval_match(*args)
     assert result == False
 
-def test_eval_match_container_measurement():
```

Comment:
I believe so as it's testing that the file's measurement field matches the acquisition's, and acquisition lost its measurement field.